docs/core/man/doveadm-acl.1.md

```diff
@@ -116,7 +116,7 @@ This command uses by default the output formatter **table**.
 ### acl add
 
 **doveadm** [*GLOBAL OPTIONS*] acl add
-  [**-u** *user* | **-A** | **-F** *file* || **\-\-no-userdb-lookup**]
+  [**-u** *user* | **-A** | **-F** *file* | **\-\-no-userdb-lookup**]
```

Comment:
I wonder how much trouble it would be to port these non-config-rewrite-related fixes that you have all over the place into main branch as well? The merges with main could become rather annoying otherwise.

Reply:
Isn't the idea to have my changes as the main branch after the review?

Reply:
They'll live in config-rewrite branch for a while. At some point config-rewrite becomes main and release-3.0 branch is created. In any case, there will be separate 3.0 and config-rewrite/3.1 branches living for a potentially long time.

Reply:
Looks like these pull requests are now to main branch. That should be changed to config-rewrite branch.

Reply:
OK, I propose the following:
- Someone - that is not me - updates the `feature/config-rewrite` branch to point to or is as close to `main` as possible,
- you - or whoever feels inclined to do so - gather the changes they wish to see from these branches,
- and I cherry-pick them into a new main-based branch, and
- I rebase these change branches unto `feature/config-rewrite` and drop the appropriate commits.

I would like to reduce annoyances - esp. those that originate from my work - to a minimum, but I also don't want my work to fade into oblivion. :open_mouth:

Reply:
I updated feature/config-rewrite to be up-to-date main now. I think we could just merge all these to config-rewrite, and then we can cherry-pick to main branch.